### PR TITLE
feature/ DAO-390 

### PR DESCRIPTION
**Pull request by sepehr2github**
- react-i18next added
- configuration files added

### CODE REVIEW

File: packages/web-app/react-i18next.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { resources, defaultNS } from './i18n.config';
+// react-i18next versions lower than 11.11.0
```
Comment (githubdoramon) [inline packages/web-app/react-i18next.d.ts]: If we are using the higher one, doesn't make sense to have the module declared twice right? You can remove this part

File: packages/web-app/tsconfig.json [hunk 2]
```diff
@@ -15,6 +15,6 @@
     "noEmit": true,
     "jsx": "react"
   },
-  "include": ["src"],
+  "include": ["src", "react-i18next.d.ts"],
```
Comment (githubdoramon) [inline packages/web-app/tsconfig.json:18]: Why is this needed?

File: packages/web-app/i18n.config.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { initReactI18next } from 'react-i18next'
+
+export const resources = {
+  en: {
+    translation: en
+  }
+} as const
+
+i18n.use(initReactI18next).init({
+  lng: 'en',
```
Comment (githubdoramon) [inline packages/web-app/i18n.config.ts]: I remember that 18next have a fallback property (so it uses the fallback translation if it does find the key. Is this still a thing?

**Review by githubdoramon — CHANGES_REQUESTED** [review]
@sepehr2github also important to add tests for validating translation keys

File: packages/web-app/i18n.config.ts (hunk 3 above)
Comment (sepehr2github) [reply packages/web-app/i18n.config.ts]: Yes, We can define fallbacks too. but because we didn't decide on some details yet. I just did basic configs.

File: packages/web-app/react-i18next.d.ts (hunk 1 above)
Comment (sepehr2github) [reply packages/web-app/react-i18next.d.ts]: Sorry, About this one!. It's happened by mistake. I removed that but I don't know why It was on my PR

**Comment by WissenIstNacht** [comment]
I would suggest a different folder structure here (also used by i18next creators in their examples):

```bash
src/
├─ i18n/
│  ├─ en/
│  │  ├─ translation.json
│  ├─ de/
│  │  ├─ translation.json
```

I would also be ok with `i18n` being called `locales`.
